File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> arr, int divisor) {
    vector<int> answer;
    
    vector<int>::iterator ptr;
    for(ptr=arr.begin(); ptr!=arr.end(); ptr++) {
        if (*ptr % divisor == 0) answer.push_back(*ptr);
    }
    
    if (answer.size()==0) {
        answer.push_back(-1);
        return answer;
    }
    else {
        sort(answer.begin(), answer.end());
        return answer;
    }
}

File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
    var answer = [];
    
    arr.map((item, index) => {
        if (item % divisor == 0) answer.push(item);
    })
    
    answer = answer.sort((a, b) => a-b);
    if (answer.length == 0) answer.push(-1);
    
    return answer;
}